prow/plank/plank_test.go

```diff
@@ -237,6 +238,7 @@ func TestSyncJenkinsJob(t *testing.T) {
 					CompletionTime: time.Now(),
 				},
 			},
+			pendingJobs:      map[string][]kube.ProwJob{},
```

Comment:
I just switched to having nil be allowed in the test case, and interpreted in the inner loop as an empty, allocated map.